web/src/portico/signup.ts

```diff
@@ -342,3 +343,102 @@ $(() => {
         }
     });
 });
+
+function get_or_set(
```

Comment:
This function is also not needed for now. Can just use `.val().trim()` while checking whether the input field overflows. A couple of if-blocks are anyways not used.

Reply:
I tried using this approach, but it only provided the static value of the input field. However, I need the dynamic values of the input. So, it didn’t work for me.  

That said, I have removed the unnecessary `if` blocks that were not being used.